packages/rum/src/apm-base.js

```diff
@@ -85,7 +82,12 @@ export default class ApmBase {
         }
       } else {
         this._disable = true
-        loggingService.info('RUM agent is inactive')
+        /**
+         * Creation of LoggingService should happen only after config update.
+         * This order gurantees that we use the proper logLevel
```

Comment:
I think it is cleaner to just call `loggingService.setLevel` in the init function instead of deciding on the order of service creation.